azure-pipelines.yml

```diff
@@ -43,23 +46,27 @@ stages:
         projectKey: '2011-fakebook-project3_posts'
         extraProperties: |
           sonar.cs.opencover.reportsPaths=$(Agent.TempDirectory)/**/coverage.opencover.xml
-          sonar.cs.vstest.reportsPaths=$(Agent.TempDirectory)/*.trx
```

Comment:
no wait, sending the trx to sonarcloud is good, not bad. was it failing?

Reply:
Re-added, assumed all pipelines would add it later together